src/assets/locales/en.js

```diff
@@ -357,7 +357,8 @@ export default {
     },
     create_plio: "Could not create Plio. Please try again",
     generic: "Something went wrong. Please try again!",
-    internet: "Please check your internet connection",
+    internet_lost: "Your device lost it's internet connection",
+    internet_found: "Your device is connected to the internet!",
```

Comment:
would it be simpler to say:

internet connection lost, internet connection restored?

Reply:
Haan makes sense, will change

Reply:
```suggestion
    internet_lost: "Internet connection lost",
    internet_found: "Internet connection restored",
```